batch/vep/build_vep_cache.sh

```diff
@@ -42,20 +42,48 @@ mkdir -p "${work_dir}"
 pushd "${work_dir}"
 readonly cache_file="${species}_vep_${release}_${assembly}.tar.gz"
 readonly ftp_base="ftp://ftp.ensembl.org/pub/release-${release}"
-readonly remote_cache="${ftp_base}/variation/VEP/${cache_file}"
-echo "Downloading ${remote_cache} ..."
-curl -O "${remote_cache}"
 
 # The fasta file name depends on the species and assembly but not the version.
 # Also the first letter of the file is capital while it is small for the actual
 # cache file (above). For example: "Homo_sapiens.GRCh38.dna.toplevel.fa.gz"
 readonly fasta_file="${species^?}.${assembly}.dna.toplevel.fa.gz"
-readonly remote_fasta="${ftp_base}/fasta/homo_sapiens/dna_index/${fasta_file}"
-echo "Downloading ${remote_fasta} and its index files ..."
-curl -O "${remote_fasta}"
-curl -O "${remote_fasta}.fai"
-curl -O "${remote_fasta}.gzi"
+if [ $species == "homo_sapiens" ] && [ $assembly == "GRCh37" ]; then
+  if [ ! `command -v samtools` ]; then
+    echo "ERROR: samtools is needed to create the .fai index."
+    echo -e "It can be installed by:\n"
```

Comment:
nit: What is the reason that you are adding empty line between these lines? Is it for clarity?